src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -222,7 +226,7 @@ const OfflineKeyShareGeneration = () => {
                             {showCopyButtonCondition && <CopyButton textCopied={textCopied} classes={classes} onClickHandler={copyToClipboard}/>}
                         </Grid>
                     }
-                    <PrimaryButton text={buttonLabel} submitFunction={submitFunctionCondition ? goToChangeOperators : goToNextPage[`${processStore.secondRegistration}`]} disable={disabledCondition()}/>
+                    <PrimaryButton text={buttonLabel} submitFunction={submitFunctionCondition ? goToChangeOperators : goToCeremonyPage} disable={disabledCondition()}/>
```

Comment:
now this button will work correctly only for DKG flow, but if we use command line (1) flow? In case when we use command line flow we have to use goToNextPage function,